Source/Core/VideoBackends/Software/SWGfx.h

```diff
@@ -3,9 +3,12 @@
 
 #pragma once
 
+#include "SWOGLWindow.h"
 #include "VideoCommon/AbstractGfx.h"
 
-class SWOGLWindow;
+// Normally we should be able to simply forward declare SWOGLWindow, but that ends up causing issues
+// with clang due to the virtual default deconstructor in AbstractGfx and the constexpr unique_ptr
+// in c++23, see https://godbolt.org/z/rPeje78e8
```

Comment:
Can't you just explicitly declare the destructor and implement it in a code file, then?

Reply:
That probably would be the better solution here. Seems like that would avoid the error and probably be more correct (seems the error only occurs if SWOGWindow deletion call occurs while the definition is still incomplete, which should be the case in this header file with just a forward declare).